evennia/contrib/game_systems/clothing/clothing.py

```diff
@@ -658,12 +658,17 @@ def func(self):
         wear_table = evtable.EvTable(border="header")
 
         carried = [obj for obj in items if not obj.db.worn]
+        # Build and populate a dict for tallying items
+        carried_sums = {obj.get_display_name(): 0 for obj in set(carried)}
```

Comment:
Thanks for the quick reply. Sorry for taking liberties with regard to the description. Happy to change it. 

I opted for (number quantity) because it'll come across as broken speech other wise, eg, five a perfect shirt. Would that be preferable? Or were you thinking a more pervasive change along the lines of Django's verbose_name_plural?

I see what you mean about the perspective change offered by self.caller. Is it preferable to have a an exception for staff or to just leave it without stacking?